dist/dfu.js

```diff
@@ -162,10 +162,11 @@
         return view;
     }
 
-    function provision(device, arrayBuffer, imageType) {
+    function provision(device, arrayBuffer, imageType, crc) {
```

Comment:
Perhaps consider putting crc ahead of imageType? Is it more likely to send a crc or send a different imageType?